requirements.txt

```diff
@@ -0,0 +1 @@
+mock
```

Comment:
Please, use `test-requirements.txt` file for libraries used only in unittest since that it is not used in the models